app/src/hooks/useXdaiBridge.tsx

```diff
@@ -1,154 +0,0 @@
-import { Zero } from 'ethers/constants'
```

Comment:
Should this be deleted?

Reply:
Well we are not using it....And we can always go back and bring it to life if needed? Though not good that I just deleted it in this pr that is not related to it...